### PR TITLE
Expose Cache loader/Writer Configuration at template

### PR DESCRIPTION
**Pull request by sreekanth-r**
This is required for #216

### CODE REVIEW

File: core/src/main/java/org/ehcache/config/CacheConfigurationBuilder.java [hunk 1]
```diff
@@ -130,4 +132,9 @@ private CacheConfigurationBuilder(final Expiry<? super K, ? super V> expiry, fin
     this.serializationProvider = serializationProvider;
     return this;
   }
+
+  public CacheConfigurationBuilder<K, V> withLoaderWriter(CacheLoaderWriter<? super K, ? super V> loaderWriter) {
```
Comment (ljacomet) [inline core/src/main/java/org/ehcache/config/CacheConfigurationBuilder.java]: Why?
It is supposed to be injected through a service

File: xml/src/main/java/org/ehcache/config/xml/XmlConfiguration.java [hunk 2]
```diff
@@ -296,6 +296,10 @@ public URL getURL() {
         .usingEvictionPrioritizer(getInstanceOfName(cacheTemplate.evictionPrioritizer(), defaultClassLoader, EvictionPrioritizer.class, Eviction.Prioritizer.class))
         .evitionVeto(getInstanceOfName(cacheTemplate.evictionVeto(), defaultClassLoader, EvictionVeto.class))
         .withExpiry(getExpiry(defaultClassLoader, parsedExpiry));
+    final String loaderWriter = cacheTemplate.loaderWriter();
```
Comment (ljacomet) [inline xml/src/main/java/org/ehcache/config/xml/XmlConfiguration.java]: This should be done the same way it is done in `org.ehcache.config.xml.XmlConfiguration#parseConfiguration`

Comment (sreekanth-r) [inline xml/src/main/java/org/ehcache/config/xml/XmlConfiguration.java]: Fixed this, removed the other Cache Configuration stuff, as loader and writer should be managed by services.

**Comment by ljacomet** [comment]
Failed build because of test failures - please have a look @sreekanth-r

**Pull request merged**